contrib/occtax/backend/occtax/migrations/e170d1902137_declare_permissions.py

```diff
@@ -0,0 +1,96 @@
+"""declare permissions
+
+Revision ID: e170d1902137
+Revises: 1b4f44762020
+Create Date: 2023-05-16 09:55:41.005029
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "e170d1902137"
+down_revision = "1b4f44762020"
+branch_labels = None
+depends_on = ("f051b88a57fd",)
+
+
+def upgrade():
+    op.execute(
+        """
+        INSERT INTO
+            gn_permissions.t_permissions_available (
+                id_module,
+                id_object,
+                id_action,
+                label,
+                scope_filter
+            )
+        SELECT
+            m.id_module,
+            o.id_object,
+            a.id_action,
+            v.label,
+            v.scope_filter
+        FROM
+            (
+                VALUES
+                     ('OCCTAX', 'ALL', 'C', True, 'Créer des relevés')
```

Comment:
À vérifier que le filtre scope ait du sens pour la création
```suggestion
                     ('OCCTAX', 'ALL', 'C', False, 'Créer des relevés')
```